modules/custom/az_migration/migrations/migrate_plus.migration.az_node_event.yml

```diff
@@ -0,0 +1,210 @@
+status: true
+
+source:
+  plugin: d7_node
+  node_type: uaqs_event
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_event
+  migration_dependencies:
+    requireed:
+      - az_event_categories
+
+process:
+  type: 
+    plugin: default_value
+    default_value: az_event
+  langcode:
+    plugin: static_map
+    bypass: true
+    source: language
+    map:
+      und: en
+  
+  title:
+    plugin: sub_process
+    source: title_field
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_subheading:
+    plugin: sub_process
+    source: field_uaqs_summary_short
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_link:
+    plugin: sub_process
+    source: field_uaqs_link
+    process:
+      delta: delta
+      uri: 
+        -
+           plugin: str_replace
+           search: '[node:url]'
+           replace: ''
+           source: url
+      title: title
+
+  field_az_location:
+    plugin: sub_process
+    source: field_uaqs_location
+    process:
+      delta: delta
+      uri: url
+      title: title
+
+  field_az_body:
+    plugin: sub_process
+    source: field_uaqs_body
+    process:
+      delta: delta
+      value: value
+      format:  
+        plugin: default_value
+        default_value: az_standard
+  
+  field_az_summary:
+    plugin: sub_process
+    source: field_uaqs_summary
+    process:
+      delta: delta
+      value: value
+      format:  
+        plugin: default_value
+        default_value: plain_text
+
+  field_az_attachments:
+    plugin: sub_process
+    source: field_uaqs_attachments
+    process:
+      delta: delta
+      target_id: 
+      - 
+        plugin: migration_lookup
+        source: fid
+        migration: 
+          - az_files
+      display: display
+      description: description
+  
+  field_az_photos:
+    plugin: sub_process
+    source: field_uaqs_photos
+    process:
+      delta: delta
+      target_id: 
+      - 
+        plugin: migration_lookup
+        source: fid
+        migration: 
+          - az_media
+  
+  field_az_contacts:
+    plugin: sub_process
+    source: field_uaqs_contacts
+    process:
+      delta: delta
+      target_id:
+        -
+          plugin: migration_lookup
+          migration: az_paragraph_contact
+          source: value
+          no_stub: true
+        -
+          plugin: extract
+          index:
+            - 0
+      target_revision_id:
+        -
+          plugin: migration_lookup
+          migration: az_paragraph_contact
+          source: value
+          no_stub: true
+        -
+          plugin: extract
+          index:
+            - 1
+
+  field_az_event_date:
+    plugin: sub_process
+    source: field_uaqs_date
+    process:
+      delta: delta
+      value:
+        -
+          plugin: format_date
+          from_format: 'Y-m-d H:i:s'
+          to_format: 'Y-m-d\TH:i:s'
+          from_timezone: 'America/Phoenix'
```

Comment:
The source should already be `UTC` from a Drupal database, I think the only change necessary is the need to display the timezone in the date so that `strtotime` uses it.

This is to prevent `strtotime` from outputting the wrong timestamp if its default timezone is not UTC.

Reply:
@tadean I will work on it as per your comment and commit somthing soon. Thanks